examples/nips17_adversarial_competition/eval_infra/code/eval_lib/image_batches.py

```diff
@@ -269,9 +269,12 @@ def __init__(self, datastore_client):
         entity_kind_batches=KIND_ADVERSARIAL_BATCH,
         entity_kind_images=KIND_ADVERSARIAL_IMAGE)
 
+  # Not possible to respect both line length and hanging indentation
```

Comment:
Would the following work:

```
def init_from_dataset_and_submissions_write_to_datastore(
    self, dataset_batches, attack_submission_ids):
  """...."""
```

This should both handle line length and intentation.

Reply:
Done, thanks. I didn't realize the first argument had to hang to enable hanging indent.